h/cli/commands/user.py

```diff
@@ -100,15 +93,13 @@ def password(ctx, username, authority, password):
 
     user = models.User.get_by_username(request.db, username, authority)
     if user is None:
-        msg = 'no user with username "{}" and authority "{}"'.format(
-            username, authority
-        )
+        msg = f'no user with username "{username}" and authority "{authority}"'
         raise click.ClickException(msg)
```

Comment:
```suggestion
        raise click.ClickException(f'no user with username "{username}" and authority "{authority}"')
```
?

Not sure if `msg` is used later?